MANIFEST.in

```diff
@@ -7,6 +7,7 @@ include NEWS
 
 include scrapy/VERSION
 include scrapy/mime.types
+include scrapy/py.typed
```

Comment:
Is this enough, no setup.py changes needed?

Reply:
I don’t think so, but I’m checking.

Reply:
In https://github.com/scrapy-plugins/scrapy-spider-metadata/pull/1, if I remove [the code to ignore the lack of type hints in Scrapy](https://github.com/scrapy-plugins/scrapy-spider-metadata/pull/1/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R58-R63), and make tox install from this branch (add `scrapy @ git+https://github.com/Gallaecio/scrapy@typing` to `testenv:types`), the error about Scrapy not having type hints goes away when running `tox -e types` (there is another error, but that’s a different story).

Reply:
Oh, it's because we have `include_package_data=True` in setup.py. 

https://setuptools.pypa.io/en/latest/userguide/datafiles.html